cmd/cl001/ac000/execute/zz_generated.runner.go

```diff
@@ -58,20 +58,14 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		if err != nil {
 			return microerror.Mask(err)
 		}
-
-		err = clients.InitTenantCluster(ctx)
-		if err != nil {
-			return microerror.Mask(err)
-		}
```

Comment:
In action `ac000` we do not have and do not need the cluster ID. It will only later be created during plan execution.